run.py

```diff
@@ -342,6 +344,17 @@ def get_tier_level(files: str) -> str:
     return "Unknown"
 
 
+def get_html_page(url: str) -> str:
+    """Returns the content of the provided link."""
+    resp = requests.get(url, verify=False)
+
+    try:
+        return resp.text
+    except BaseException as be:  # noqa
+        log.debug(f"Failed to retrieve contents at {url}")
+        return "Unknown"
```

Comment:
Can we set `build type - upstream | rc | releases` rather than `unknown` which will ease debugging about build information?

Reply:
In case of `upstream | released`, this method would not be called as we don't check for versions when `build` has the those values.

Reply:
> 

Exactly, this method won't be called and ceph versions would be empty without being known as released or upstream.

Reply:
so would it make sense to add else part to set these ceph and ceph-ansible versions.?

Reply:
As discussed, we will pick the changes via a new PR specific to `build-type`